frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-queries/index.md

```diff
@@ -0,0 +1,232 @@
+# UniConfig Queries
+
+This module is responsible for execution of queries on the configuration of some device, template, UniStore node,
+or snapshot.
+
+## RPC query-config
+
+UniConfig exposes filtering and selection API using RPC 'query-config'. Filtering and selection of configuration
+is done only on the database side - UniConfig receives already narrowed configuration with only selected data.
+Since query is evaluated by the database, this feature is works only with already committed data (operational data).
+
+The following sequence diagram captures the whole process of RPC execution in detail.
+
+![Execution of RPC query-config](puml-RPC_query_config.svg)
+
+### RPC input fields
+
+- **topology-id**: Identifier of network-topology/topology list entry. Currently, supported topologies, under which
+  this RPC can be used, are: 'uniconfig', 'templates', 'unistore', and snapshot topologies.
+- **node-id**: Identifier of specific network-topology/node list entry which configuration is filtered using specified
```

Comment:
Identifier of specific network-topology/node list entry ~~which~~ **whose** configuration

Reply:
fixed in next commit